templates/help_request/list.html

```diff
@@ -143,20 +144,20 @@
     <script type="text/template" id="table-empty-template">
       {% verbatim %}
 	<div class="media">
-	    <h1>No se encuentran pedidos de ayuda en la ubicación actual.</h1>
+	    <h1>{% trans "Can't find requests at your current location" %}</h1>
 	</div>
       {% endverbatim %}
     </script>
 
     <script type="text/template" id="table-paginator-template">
-          {% verbatim %}
+	    {% verbatim %}
         	<div class="row table-paginator">
         	    <div class="column is-5">
-        		  Mostrando &lt; Página {{currentPage}} de {{totalPages}} &gt;
+			{% trans "Displaying" %} &lt; {% trans "Page" %} {{currentPage}} {% trans "of" %} {{totalPages}} &gt;
```

Comment:
@melizeche @jorgeramirez need to figure out how to move these user-facing strings out of the `verbatim` tag... otherwise it won't really get translated... ideas?

Reply:
Something that we could try is to "narrow" the `verbatim` tag. For example,

```
{% verbatim %}  {{currentPage}}  {% endverbatim %}
```
and something similar for the rest of the stuff in double braces. It is definitely not a fancy solution tough.